tech-far-hub/package.json

```diff
@@ -13,11 +13,16 @@
     "build": "gatsby build",
     "serve": "gatsby serve",
     "clean": "gatsby clean",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --noEmit",
```

Comment:
Nice! I just added this myself to the new repo.. although stuff is failing.. which means I might have to patch it